File: 1720-crawler-log-folder/crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int cur=0;
        int n=logs.size();
        for(int i=0;i<n;i++){
            int m=logs[i].length();
            if(m==3 && logs[i][0]=='.' && logs[i][1]=='.'){
                if(cur!=0){
                    cur--;
                }
            }
            else if(m==2 && logs[i][0]!='.' || m>=3){
                cur++;
            }
        }
        return cur;
    }
};

File: 1720-crawler-log-folder/crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        n=len(logs)
        cur=0;
        for i in range(n):
            m=len(logs[i])
            if m==3 and logs[i][0]=='.' and logs[i][1]=='.':
                if(cur!=0):
                    cur-=1;
                
            elif m==2 and logs[i][0]!='.' or m>=3:
                cur+=1;
        return cur